vpc/service/service.go

```diff
@@ -276,14 +273,6 @@ func Run(ctx context.Context, config *Config) error {
 			"hostname":  keyHostname,
 			"createdAt": keyCreatedAt,
 		}).Debug("Found matching public key for private key")
-
-		row := config.DB.QueryRowContext(ctx, "SELECT nextval('worker_id')")
```

Comment:
Why don't we need this code again?

Reply:
Because this was to work around the fact that if you didn't assign a unicast address, router advertisements were broken.